### PR TITLE
Add support for Adobe Analytics, Adobe Target, and Adobe Audience Manager data layers in variable data elements.

### PR DESCRIPTION
**Pull request by jonsnyder**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.
- [ ] I've updated the schema in extension.json or no changes are necessary.
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/view/actions/updateVariable.jsx [hunk 1]
```diff
@@ -82,6 +86,19 @@ const getInitialFormStateFromDataElement = async ({
       });
     }
   }
+  if (dataElement?.settings?.data) {
+    const schema = generateSchemaFromSolutions(dataElement.settings.data);
```
Comment (jonsnyder) [inline src/view/actions/updateVariable.jsx:90]: From looking at the manifest, this should be "dataElement.settings.solutions".

File: src/view/actions/updateVariable.jsx [hunk 2]
```diff
@@ -128,6 +143,14 @@ const getInitialValues = context => async ({ initInfo }) => {
   ) {
     dataElement = dataElementsFirstPage[0];
   }
+
+  if (bypass && dataElement) {
+    dataElement.settings = {
+      cacheId: "e4d4092c-0f95-42fd-9d51-b40328fee5eb",
+      data: ["target", "analytics"]
```
Comment (jonsnyder) [inline src/view/actions/updateVariable.jsx:150]: This should be solutions too.

File: src/view/actions/updateVariable.jsx (hunk 1 above)
Comment (dompuiu) [reply src/view/actions/updateVariable.jsx:90]: Good catch.

**Comment by jonsnyder** [comment]
Closed in favor of #383

**Pull request closed without merging**